tests/lib/rules/define-macros-order.js

```diff
@@ -254,6 +273,22 @@ tester.run('define-macros-order', rule, {
           order: ['defineModel', 'defineSlots']
         }
       ]
+    },
+    {
+      filename: 'test.vue',
+      code: `
+        <script setup>
+        const page = definePage()
+
+        const first = defineModel('first')
+          const second = defineModel('second')
```

Comment:
Even if it's just test code, let's format it properly:

```suggestion
        const first = defineModel('first')
        const second = defineModel('second')
```